bindings/python/tests/cancellation_timeout_tests.py

```diff
@@ -518,7 +518,6 @@ def txn11(tr):
         for i in range(2):
             tr.options.set_timeout(1500)
             tr.set_read_version(0x7ffffffffffffff0)
-            x = tr[b'foo']
```

Comment:
It's possible removing this line breaks the test. Possibly it could be changed to `tr[b'foo'].wait()` though.